### PR TITLE
Combine pseudo class name selector if ie is off

### PR DESCRIPTION
**Pull request by roderickhsiao**
@src-code @thierryk 

Our default setting for Atomizer is `ie: false`, I think it is safe to combine pseudo class names selector which we previously separate them to support `< IE 8` (probably no one cares much for now).

This PR passes ie options to the `combineSelectors` function and bypass those exception rule for old IE handling.

### CODE REVIEW

File: tests/lib/jss.js [hunk 1]
```diff
@@ -435,5 +435,41 @@ describe('JSS', function () {
             ].join('\n');
             expect(result).to.equal(expected);
         });
+        it('should combine pseudo selectors if ie options is off', function () {
```
Comment (src-code) [inline tests/lib/jss.js]: This test seems to test both cases (on and off), so perhaps it should be broken up into two tests to be more clear what we're testing?

Comment (roderickhsiao) [inline tests/lib/jss.js]: sure

**Comment by roderickhsiao** [comment]
btw, Happy July 4 🇺🇸

**Comment by src-code** [comment]
Worth noting, this has the potential to be a breaking change if people were for some reason supporting old IE but not setting the `ie` config to true.

**Comment by roderickhsiao** [comment]
thats true, the assumption of of people supporting old IE will always set `ie: true` probably too strong.

**Comment by roderickhsiao** [comment]
adding logic to check `undefined` and default to true for backward compatibility

**Comment by roderickhsiao** [comment]
added logic to check `undefined` and default to `true` for backward compatibility, not my favorite, but let me know if you have other thoughts :)

**Pull request closed without merging**